src/components/DateRangePicker.js

```diff
@@ -28,6 +28,7 @@ class DateRangePicker extends React.Component {
     minimumDate: PropTypes.number,
     onClose: PropTypes.func,
     onDateSelect: PropTypes.func,
+    optionsWrapperStyle: PropTypes.object,
```

Comment:
@bsbeeks we're moving away from using individual props for updating specific styles. Can you please change this to use the `styles` prop pattern? Here's an example https://github.com/mxenabled/mx-react-components/blob/master/src/components/SimpleSelect.js#L156

Reply:
Love it, I'll make the change.